api/app/src/command/medical/document/document-download.ts

```diff
@@ -1,9 +1,78 @@
+import { makeLambdaClient } from "../../../external/aws/lambda";
 import { makeS3Client } from "../../../external/aws/s3";
 import { Config } from "../../../shared/config";
 
+const lambdaClient = makeLambdaClient();
 const s3client = makeS3Client();
 
-export const downloadDocument = async ({ fileName }: { fileName: string }): Promise<string> => {
+export const downloadDocument = async ({
+  fileName,
+  conversionType,
+}: {
+  fileName: string;
+  conversionType?: string;
+}): Promise<string> => {
+  if (conversionType === "html") {
+    const url = await getConversionUrl({ fileName, conversionType });
+    return url;
+  } else if (conversionType === "pdf") {
```

Comment:
nit: could put the `conversionType` in a literal

Reply:
If we make a union type this would be fine, though 😉